stable/drupal/values.yaml

```diff
@@ -39,6 +39,12 @@ allowEmptyPassword: yes
 ## MariaDB chart configuration
 ##
 mariadb:
+  ## Whether to use the database specified as a requirement or not. For example, to configure the chart with an existing database server.
+  enabled: true
+
+  ## MariaDB host
+  # mariadbHost:
```

Comment:
`mariadb.mariadbHost` is a bit redundant, just `mariadb.host` is fine.

We should add a comment that this option is only used when `mariadb.enabled = false` so it doesn't get confused for an option in the mariadb chart itself.